go.mod

```diff
@@ -4,6 +4,7 @@ go 1.16
 
 require (
 	github.com/blang/semver/v4 v4.0.0
+	github.com/ghodss/yaml v1.0.0
```

Comment:
Please do not use this YAML package. It's unmaintained and stuff. Use https://github.com/kubernetes-sigs/yaml instead. Same interface, and maintained.

Reply:
Thanks! I will update here!

FWIW, I just sent a PR for `eventing-kafka`, which uses this package instead as well

Will do the same for the `knative/operator` as well